Dockerfile

```diff
@@ -1,23 +1,22 @@
 FROM python:3.8-slim as base
 
-FROM base as builder
 # Any python libraries that require system libraries to be installed will likely
 # need the following packages in order to build
-RUN apt-get update && apt-get install -y build-essential git
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    apt-get install -y build-essential git && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
I've not seen people thin out their docker images after installing via apt-get. Nice.